src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1301,7 +1301,7 @@ private void invalidateLoadFlowStatusOnAllNodes(UUID studyUuid) {
         loadflowService.invalidateLoadFlowStatus(rootNetworkNodeInfoService.getComputationResultUuids(studyUuid, LOAD_FLOW));
     }
 
-    private void invalidateNodeTreeWithLoadFlowResults(UUID studyUuid) {
+    public void invalidateNodeTreeWithLoadFlowResults(UUID studyUuid) {
```

Comment:
This punlic method is called by a private method !

Reply:
I agree, but what it the best workaround here

Reply:
fixed